stack/keycloak/migration/add-authorities-basisdaten-referendumvorlagen.yml

```
@@ -0,0 +1,66 @@
+id: add authorities basisdaten referendumvorlagen
+author: MrSebastian
+realm: ${SSO_REALM}
+changes:
+  - addRole:
+      name: Basisdaten_BUSINESSACTION_GetReferendumvorlagen
+      clientRole: true
+      clientId: ${SSO_CLIENT_ID}
+  - assignRoleToGroup:
+      group: allBasisdatenAuthorities
+      role: Basisdaten_BUSINESSACTION_GetReferendumvorlagen
+      clientId: ${SSO_CLIENT_ID}
+
+  - addRole:
+      name: Basisdaten_READ_Referendumvorlagen
+      clientRole: true
+      clientId: ${SSO_CLIENT_ID}
+  - assignRoleToGroup:
+      group: allBasisdatenAuthorities
+      role: Basisdaten_READ_Referendumvorlagen
+      clientId: ${SSO_CLIENT_ID}
+
+  - addRole:
+      name: Basisdaten_WRITE_Referendumvorlagen
+      clientRole: true
+      clientId: ${SSO_CLIENT_ID}
+  - assignRoleToGroup:
+      group: allBasisdatenAuthorities
+      role: Basisdaten_WRITE_Referendumvorlagen
+      clientId: ${SSO_CLIENT_ID}
+
+  - addRole:
+      name: Basisdaten_DELETE_Referendumvorlagen
```

```suggestion
      name: Basisdaten_DELETE_WLSReferendumvorschlaege
```
so wäre es im Altsystem

---

besprochen:
- neues Rechte ist lesbarer
- Skripte zur DB-Initialisierung müssen neu geschrieben werden was ein vertretbarer Aufwand ist